### ISSUE TITLE
Warrior's Battle Shouts don't follow the "a more powerful spell" mechanics.

### ISSUE THREAD

**Issue by Cabro** [LightsHope/server#589]
And they should.

**Comment by ohganot** [comment]
Pretty sure I have tried that and I couldn't overwrite higher Battle Shouts.

**Comment by Cabro** [comment]
Also pretty sure i could overwrite a battle shout with T2 bonus.

**Comment by ohganot** [comment]
It's very likely that a tank who's using 3 pieces of T2 isn't specced into Imp. Battle Shout, so it would be normal for a fury warrior to be able to overwrite that with Imp. Battle Shout.

Would need further testing

**Comment by Cabro** [comment]
No. I tested myself. Used t2 bonus, then unequiped t2 and shouted, and i overwrote the previous shout.

**Comment by GuybrushGit** [comment]
With or without t2 its the same spell. Its supposed to refresh duration.

**Comment by Cabro** [comment]
But the Battle Shout with T2 bonus has 30 more AP than the base one. You have http://db.vanillagaming.org/?spell=23563 added on top of the Battle Shout. It shouldn't be overwritten.

**Comment by knasen88** [comment]
https://forum.elysium-project.org/topic/48113-discussion-a-more-powerful-spell-is-already-active-and-how-it-should-be-implemented-for-the-elysium-core/

there is proof inside this thread that the "a more powerful spell is active" mechanic exists for vanilla and nothing that says it should not also be true for Battle Shout.

**Comment by GuybrushGit** [comment]
Is there any specific example of one spell not overwriting the exact same spell id, due to talents or +heal modifiers?

**Comment by Cabro** [comment]
The 5/5 T2.5 priest bonus should act like a more powerful renew than the normal one with the same id.
